### PR TITLE
OKTA-360964: release v1.0.0

### PR DESCRIPTION
**Pull request by bryanapellanes-okta**
- updated release related files
- updated dependency versions
- updated README
- updated CHANGELOG

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,12 @@
 # Changelog
 Running changelog of releases since `1.0.0-beta01`
 
-## v1.0.0-beta03
+## v1.0.0
```
Comment (bryanapellanes-okta) [inline CHANGELOG.md:4]: Skipped beta03 to focus on implementing all of v1

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
 
 ### Features
 
-- Added OktaContext.AuthenticationFailed event
+- Added OktaContext convenience methods and events
```
Comment (laura-rodriguez) [inline CHANGELOG.md:8]: Can you please add more details on what new methods and events were introduced in this version? Same for Okta state manager.

File: CHANGELOG.md (hunk 1 above)
Comment (laura-rodriguez) [reply CHANGELOG.md:4]: I suggest adding a short description indicating that this is the first stable version 🎉

File: Okta.Xamarin/Okta.Xamarin.iOS/Okta.Xamarin.iOS.csproj [hunk 3]
```diff
@@ -99,8 +99,8 @@
     <PackageReference Include="System.Net.Http">
```
Comment (laura-rodriguez) [inline Okta.Xamarin/Okta.Xamarin.iOS/Okta.Xamarin.iOS.csproj:99]: Should we add the project version here and in the other projects (Xamarin and Android)?

**Review by laura-rodriguez — APPROVED** [review]
Some minor comments, then LGTM 🚀 .

File: Okta.Xamarin/Okta.Xamarin.iOS/Okta.Xamarin.iOS.csproj (hunk 3 above)
Comment (bryanapellanes-okta) [reply Okta.Xamarin/Okta.Xamarin.iOS/Okta.Xamarin.iOS.csproj:99]: Omitting the version should cause it to get the latest, or whatever is available.

Comment (bryanapellanes-okta) [reply Okta.Xamarin/Okta.Xamarin.iOS/Okta.Xamarin.iOS.csproj:99]: To add a little more detail, I believe that the System.Net.Http component is wired in a specific way on iOS based on other iOS project settings; the core (or all) of it's functionality is delegated to a native component in iOS (I believe).

**Review by andriizhegurov-okta — APPROVED** [review]
👍

**Pull request merged**